kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/KieServerDeployer.java

```diff
@@ -61,24 +62,19 @@ public static void buildAndDeployMavenProjectFromResource(String resourcePath) {
     }
 
     public static void buildAndDeployMavenProject(String basedir) {
-        // need to backup (and later restore) the current class loader, because the Maven/Plexus does some classloader
-        // magic which then results in CNFE in RestEasy client
         // run the Maven build which will create the kjar. The kjar is then either installed or deployed to local and
         // remote repo
         logger.debug("Building and deploying Maven project from basedir '{}'.", basedir);
-        ClassLoader classLoaderBak = Thread.currentThread().getContextClassLoader();
-        System.setProperty("maven.multiModuleProjectDirectory", basedir); // required by MavenCli 3.3.0+
-        MavenCli cli = new MavenCli();
         List<String> mvnArgs;
         if (TestConfig.isLocalServer()) {
             // just install into local repository when running the local server. Deploying to remote repo will fail
             // if the repo does not exist.
 
             // wrapping explicitly in ArrayList as we may need to update the list later (and Arrays.toList() returns
             // just read-only list)
-            mvnArgs = new ArrayList<String>(Arrays.asList("-B", "clean", "install"));
+            mvnArgs = new ArrayList<>(Arrays.asList("mvn", "--quiet", "clean", "install"));
```

Comment:
I usually have problem with embedded maven not showing error messages.
Can this --quiet worsen the situation?

Reply:
The errors should be still shown. From the maven help:
`-q,--quiet                             Quiet output - only show errors`

I have tried that manually, the error is shown if any error happens.

Reply:
Ok perfect